packages/web/src/components/createCell.tsx

```diff
@@ -104,14 +106,13 @@ export type CellSuccessData<TData = any> = Omit<Guaranteed<TData>, '__typename'>
 export type CellSuccessProps<
   TData = any,
   TVariables extends OperationVariables = any
-> = Partial<
-  Omit<
+> = Partial<{
```

Comment:
Sorry I didn't catch this earlier, but I wonder if `Partial<>` isn't in the wrong spot now. I could be wrong here, so please validate my claims. But I *think* what you're saying now is that `queryResult` might or might not be present. And `updating` might or might not be present.

Basically this:
```ts
type CellSuccessProps = {
  queryResult?: { ... }
  updating?: boolean
}
```

Whereas what we had before was everything "inside" `queryResult` being optional.

So, maybe what we need is something like
```ts
type CellSuccessProps = {
  queryResult: Partial<{ ... }>
  updating: boolean
}
```

I removed the `?` after both `queryResult` and `updating`. I'm not sure if for example `updating` can be `undefined`, and if so, how React will treat `<Success updating={undefined}>`. If it can be `undefined`, and react then doesn't include it in the props for `<Success>`, then we need to add the `?`.

Reply:
Yeah I thought about this briefly. We had `Partial` wrapping the whole type before so pretty much everything except for the data comes back as being possible undefined:

<img width="1522" alt="image" src="https://user-images.githubusercontent.com/32992335/235372491-20119f1a-6f97-47c7-9aee-1e94b977cd05.png">

Including `updating`:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/32992335/235372518-a4a1936f-1847-4560-ae02-0dc1e1339018.png">

It makes sense to make `queryResult`'s contents Partial but not the prop itself, but making `updating` not possibly undefined would be a change I haven't considered yet. Open to anything though I just haven't through it through, was just trying to fix the bug

Reply:
With the typing the way it is I have to pass `queryResult` like this in the type test now; maybe it's not a big deal since it'll be passed by the Cell to Success, etc automatically: https://github.com/redwoodjs/redwood/pull/8175/commits/640461954fd2f1bbe1f7b18743c6311e7866c35f

Reply:
> Open to anything though I just haven't through it through, was just trying to fix the bug

You're right. Let's focus on the actual bug! 🙂

Reply:
> With the typing the way it is I have to pass `queryResult` like this in the type test now

I think that's correct. The "input" to a cell should/will have `queryResult`.